README.md

```diff
@@ -6,6 +6,8 @@
 
 <h1 align="center">Halstack Design System</h1>
 
+[![Check Links](https://github.com/dxc-technology/halstack-react/actions/workflows/check-links.yml/badge.svg)](https://github.com/dxc-technology/halstack-react/actions/workflows/check-links.yml)
```

Comment:
These are internal checks, we must try to appear as solid and ‘perfect’ as possible. I don't see the value for the users to see this badge, and we as developers always go to the actions so...